docker-for-aws/index.md

```diff
@@ -38,6 +38,67 @@ using CloudFormation. For more about stable and beta channels, see the
   </tr>
 </table>
 
+## Deployment options
+
+There are two ways to deploy Docker for AWS, you can install inside of a pre-existing VPC, or you can have Docker for AWS create the VPC and everything that it needs. We recommend the second option, since it allows us to setup the environment the way in the best way possible. If you have to install in an existing VPC, that is fine, but it will require a little more work.
```

Comment:
Do you mind changing the text `inside of a pre-existing VPC` to `with a pre-existing VPC` ?